polaris-react/src/components/TopBar/components/Menu/Menu.tsx

```diff
@@ -32,8 +37,13 @@ export interface MenuProps {
   customWidth?: string;
   /** A boolean property indicating whether the menu is being used as a user menu */
   userMenu?: boolean;
+  /** Whether to indent the menu items, or not */
+  indent?: boolean;
 }
 
+const STORE_LIST_SECTION_ID = 'Stores';
+const OTHER_STORES_ID = 'otherStores';
```

Comment:
The `Switch Organizations` button has an `id` of otherStores. Assume variables with this name to be that item.

Reply:
Having these ids hardcoded is a bit strange IMO. The Polaris component shouldn't care about the concept of stores.

Reply:
I can understand eventually replacing `otherStores` (which is used to avoid the indentation for now) with item specific props, but how can we extract `Stores` without specifying it's ID? 
If we choose the first item without this ID, we risk affecting sections without the store list. If you can think of a better way to handle that, please let me know.